spec/validate-body-decorator.spec.ts

```diff
@@ -0,0 +1,156 @@
+import {
+	assertEquals,
+	assertExists,
+} from 'https://deno.land/std@0.135.0/testing/asserts.ts';
+import { DanetApplication } from '../src/mod.ts';
+import { Injectable } from '../src/injector/injectable/mod.ts';
+import { Module } from '../src/module/mod.ts';
+import { Body, Controller, Get, Post } from '../src/router/controller/mod.ts';
+
+import { IsNumber, IsString } from 'https://deno.land/x/validatte/mod.ts';
+
+// Utils ---------------
+function jsonWithMessage(msg: string) {
+	return ({ message: msg });
+}
+async function fetchWithBody(url: string, body: any) {
+	return fetch(url, {
+		method: 'POST',
+		headers: {
+			'Content-Type': 'application/json',
+		},
+		body: JSON.stringify(body),
+	});
+}
+// ---------------------
+
+class DTO {
+	@IsString()
+	name!: string;
+
+	@IsNumber()
+	age!: number;
+}
+
+@Controller('/test')
+class AppController {
+	constructor() {}
+
+	@Get('')
+	justSayHello() {
+		return jsonWithMessage('Hello');
+	}
+
+	@Post('')
+	sayHelloToHim1(@Body() body: DTO) {
+		return jsonWithMessage(`Hello ${body.name}`);
+	}
+
+	@Post('/body-with-prop')
+	sayHelloToHim2(@Body('name') body: DTO) {
```

Comment:
This argument should be of type `string`, not `DTO`

Reply:
Sure! I missed change it before open the PR ksksks